tests/ro-crate-metadata.json

```diff
@@ -0,0 +1,457 @@
+        }
+    ],
+    "@graph": [
+        {
+            "@id": "./",
+            "@type": "Dataset",
+            "conformsTo": [
+                {
+                    "@id": "https://w3id.org/ro/crate/1.1"
+                },
+                {
+                    "@id": "https://w3id.org/ro/wfrun/workflow/0.1"
+                },
+                {
+                    "@id": "https://w3id.org/workflowhub/workflow-ro-crate/1.0"
+                }
+            ],
+            "datePublished": "2023-06-08T09:25:04+00:00",
+            "hasPart": [
+                {
+                    "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimming_and_qc.cwl"
+                },
+                {
+                    "@id": "exe/fastqc.cwl"
+                },
+                {
+                    "@id": "exe/trimmomatic_pe.cwl"
+                },
+                {
+                    "@id": "exe/ERR034597_1.small.fq.gz"
+                },
+                {
+                    "@id": "exe/ERR034597_2.small.fq.gz"
+                },
+                {
+                    "@id": "outputs/ERR034597_1.small.fq.trimmed.1P.fq"
+                },
+                {
+                    "@id": "outputs/ERR034597_1.small.fq.trimmed.1U.fq"
+                },
+                {
+                    "@id": "outputs/ERR034597_1.small.fq.trimmed.2P.fq"
+                },
+                {
+                    "@id": "outputs/ERR034597_1.small.fq.trimmed.2U.fq"
+                },
+                {
+                    "@id": "outputs/ERR034597_1.small_fastqc.html"
+                },
+                {
+                    "@id": "outputs/ERR034597_2.small_fastqc.html"
+                },
+                {
+                    "@id": "stdout.log"
+                },
+                {
+                    "@id": "stderr.log"
+                }
+            ],
+            "mainEntity": {
+                "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimming_and_qc.cwl"
+            }
+        },
+        {
+            "@id": "ro-crate-metadata.json",
+            "@type": "CreativeWork",
+            "about": {
+                "@id": "./"
+            },
+            "conformsTo": [
+                {
+                    "@id": "https://w3id.org/ro/crate/1.1"
+                },
+                {
+                    "@id": "https://w3id.org/workflowhub/workflow-ro-crate/1.0"
+                }
+            ]
+        },
+        {
+            "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimming_and_qc.cwl",
+            "@type": [
+                "File",
+                "SoftwareSourceCode",
+                "ComputationalWorkflow"
+            ],
+            "attachment": [
```

Comment:
`attachment` is not in the context. What is being tried to express here? If those are CWL tools or subworkflows, they should be linked to via `hasPart`.

Reply:
This might have been marked as resolved by mistake. The issue is still present.